alns/accept/AdaptiveThreshold.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+
+
+class AdaptiveThreshold:
+    """
```

Comment:
Could you streamline the docstrings a bit more? The references and parameters sections don't really work out yet.